mods/nandonaltseaonsmod.yaml

```diff
@@ -0,0 +1,16 @@
+format: 1
+name: Seasons Mod
+authors: [Nandonalt]
+desc: |
+  This is the Seasons Mod. A mod that adds the four seasons to Minecraft, each one with its own features(and danger!)
+  This requires ModLoader along with ScotTools or ShockAhPI.
```

Comment:
This isn't really an issue per-say, but just something to be aware of. Since the descriptions are parsed with markdown, these two separate lines will be combined into a single paragraph unless you put a blank line between them.